website/search/util.py

```diff
@@ -114,7 +117,10 @@ def create_atom_feed(name, data, query, size, start, url, to_atom):
     )
 
     for doc in data:
-        feed.add(**to_atom(doc))
+        try:
+            feed.add(**to_atom(doc))
+        except ValueError as e:
+            logger.exception('Atom feed error for source {}: {}'.format(doc.get('source'), e))
```

Comment:
logger.exception takes an `Exception`, not a string. See the change here: https://github.com/CenterForOpenScience/osf.io/commit/ffabb5a6d65e843f0ab583628bacd40ce3506354